_observing-your-data/trace/ta-dashboards.md

```diff
@@ -10,16 +10,106 @@ redirect_from:
 
 # Trace Analytics OpenSearch Dashboards plugin
 
-The Trace Analytics plugin for OpenSearch Dashboards provides at-a-glance visibility into your application performance, along with the ability to drill down on individual traces. For installation instructions, see [Standalone OpenSearch Dashboards plugin install]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/).
+The Trace Analytics plugin offers at-a-glance visibility into application performance based on OpenTelemetry protocol data.
+Introduced 2.15
```

Comment:
Should be kept as the original version IMO (Introduced in OpenSearch 1.0). We just updated the existing plugin in 2.15

Reply:
Doc is reformatted, so this issue is resolved.